Readme.md

```diff
@@ -92,88 +81,84 @@ or in common to all routes.  Steps in common can be either before or after the
+Canonical server using [http]:
+
+    var http = require('http');
+    var querystring = require('querystring');
+    var server = http.createServer(function(req, res) {
+        req.data = "";
+        req.on('data', function(chunk) { req.data += chunk; });
+        req.on('end', function() {
+            var url = req.url, qs = url.indexOf('?');
+            if (qs >= 0) req.params = querystring.parse(url.slice(qs+1));
+            res.writeHead(200, {'Content-Type': 'application/json'});
+            res.end(JSON.stringify(req.params));
         });
-        server.listen(1337, '127.0.0.1');
-        // 17.6k/s  wrk -d8s -t2 -c8 'http://localhost:1337/echo?a=1'
+    });
+    server.listen(1337, '127.0.0.1');
+    // 17.6k/s  wrk -d8s -t2 -c8 'http://localhost:1337/echo?a=1'
 
 With restiq:
 
-        var Restiq = require('restiq');
-        var app = Restiq.createServer({readImmediate: 0});
-        app.addStep(Restiq.mw.parseQueryParams);
-        app.addRoute('GET', '/echo', [
-            function(req, res, next) {
-                res.writeHeader(200, {'Content-Type': 'application/json'});
-                res.end(JSON.stringify(req.params));
-                next();
-            }
-        ]);
-        app.listen(1337);
-        // 20.9k/s  wrk -d8s -t2 -c8 'http://localhost:1337/echo?a=1'
-
-With restify:
-
-        var restify = require('restify');
-        var app = restify.createServer();
-        app.use(restify.queryParser());
-        app.get('/echo', function(req, res, next) {
-            res.send(200, req.params);
+    var Restiq = require('restiq');
+    var app = Restiq.createServer({readImmediate: 0});
+    app.addStep(Restiq.mw.parseQueryParams);
+    app.addRoute('GET', '/echo', [
+        function(req, res, next) {
+            res.writeHeader(200, {'Content-Type': 'application/json'});
+            res.end(JSON.stringify(req.params));
             next();
-        });
-        app.listen(1337);
-        // 4.6k/s  wrk -d8s -t2 -c8 'http://localhost:1337/echo?a=1'
+        }
+    ]);
+    app.listen(1337);
+    // 20.9k/s  wrk -d8s -t2 -c8 'http://localhost:1337/echo?a=1'
+
+With [restify]:
+
+    var restify = require('restify');
+    var app = restify.createServer();
+    app.use(restify.queryParser());
+    app.get('/echo', function(req, res, next) {
+        res.send(200, req.params);
+        next();
+    });
+    app.listen(1337);
+    // 4.6k/s  wrk -d8s -t2 -c8 'http://localhost:1337/echo?a=1'
 
 Change just the first two lines to run it under restiq:
 
-        var restify = require('restiq');
-        var app = restify.createServer({restify: true});
-        // ...
-        // 16.2k/s  wrk -d8s -t2 -c8 'http://localhost:1337/echo?a=1'
+    var restify = require('restiq');
+    var app = restify.createServer({restify: true});
+    // ...
+    // 16.2k/s  wrk -d8s -t2 -c8 'http://localhost:1337/echo?a=1'
 
 
 Methods
 -------
 
-### Restiq( options )
-### Restiq.createServer( options )
+### `restiq( options )`
```

Comment:
While I prefer the code in titles (since this is code-ish), I will happily return them to non-code bits of text. I defer to following your style. :-)

bb990a6